### DISCUSSION TITLE
Is it necessary to install packages on every compile?

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#2344] [Q&A] [answered]
I was just wondering if it really is important for reflex to install base frontend packages on every small change i make to my code?
Or is there something wrong with my configuration or installation?
It just slows down the hot reload on every small change.

**Comment by masenf** [comment]
Is it actually doing install work? I thought that reflex will check whether packages need to be installed using `bun`, but if there are no changes to what's installed, then it should mostly be a no-op. How long are you seeing this step take?

**Comment by masenf** [comment]
On what platforms have you experienced this issue? I'm running on my macOS 14.2.1 dev environment and I don't see any package installation happening for simple changes.

**Reply by NanoscaleSimulations** [reply]
I am on Windows 11, using python 3.11, in a conda environment and reflex 0.3.7.
It seems like it does actually reinstall all packages, because it takes too long if you ask me. Somewhere between 2-5 seconds, sometimes longer.

**Comment by NanoscaleSimulations** [comment]
Any fix on this?

**Reply by picklelo** [reply, marked as answer]
We're investigating this issue this week, hoping to fix this in a release soon.

**Reply by picklelo** [reply]
https://github.com/reflex-dev/reflex/pull/2400

**Reply by NanoscaleSimulations** [reply]
Excellent :) appreciate it
